docs/resources/waf_domain.md

```diff
@@ -121,18 +129,18 @@ The following arguments are supported:
 
 * `http2_enable` - (Optional, Bool) Specifies whether to use the http2 protocol.
   This field is only used for communication between clients and WAF.
+  Defaults to **false**.
   Things to note when using this field are as follows:
-  + There must be at least one server configuration with client protocol set to `HTTPS`, or this configuration is unable
+  + There must be at least one server configuration with client protocol set to **HTTPS**, or this configuration is unable
     to work.
-  + This field cannot not work if the client supports TLS 1.3.
-  + This field can work only when the client supports TLS 1.2 or earlier versions.
+  + This field cannot not work if the client supports **TLS 1.3**.
+  + This field can work only when the client supports **TLS 1.2** or earlier versions.
   + If you want to use HTTP/2 forwarding, use a dedicated WAF instance.
 
-  Defaults to **false**.
-
 * `ipv6_enable` - (Optional, Bool) Specifies whether IPv6 protection is enabled.
-  Enable IPv6 protection if the domain name is accessible using an IPv6 address.
-  After you enable it, WAF assigns an IPv6 address to the domain name.
+  As long as the `server.type` value is **ipv6**, the value of `ipv6_enable` must be
+  set to **true** and cannot be modified to **false** again, that is,
+  ipv6 protection cannot be turned off.
   Defaults to **false**.
```

Comment:
Specifies whether IPv6 protection is enabled. Enable IPv6 protection if the domain name is accessible using an IPv6 address. After you enable it, WAF assigns an IPv6 address to the domain name.
This field must be set to **true** when "server" contains a value of type **ipv6**. Defaults to **false**.